CONTRIBUTING.md

```diff
@@ -1,28 +1,55 @@
+# Contributing to this repository
 
-If you want to submit a pull request to fix a bug or enhance an existing feature, please first open an issue and link to that issue when you submit your pull request.  
+We welcome your contributions! There are multiple ways to contribute.
 
-If you have any questions about a possible submission, feel free to open an issue too.
+## Opening issues
 
-## [Contributing to the WebLogic Deploy repository](http://github.com/oracle/wls-deploy/blob/master/CONTRIBUTING.md)
-Pull requests can be made under [The Oracle Contributor Agreement](https://www.oracle.com/technetwork/community/oca-486395.html) (OCA).
+For bugs or enhancement requests, please file a GitHub issue unless it's
+security related. When filing a bug remember that the better written the bug is,
+the more likely it is to be fixed. If you think you've found a security
+vulnerability, do not raise a GitHub issue and follow the instructions in our
+[security policy](./SECURITY.md).
 
-For pull requests to be accepted, the bottom of your commit message must have the following line using your name and e-mail address as it appears in the OCA Signatories list.
+## Contributing code
 
-    Signed-off-by: Your Name <you@example.org>
+We welcome your code contributions. Before submitting code via a pull request,
+you will need to have signed the [Oracle Contributor Agreement][OCA] (OCA) and
+your commits need to include the following line using the name and e-mail
+address you used to sign the OCA:
 
-This can be automatically added to pull requests by committing with:
+```text
+Signed-off-by: Your Name <you@example.org>
+```
 
-    git commit --signoff
+This can be automatically added to pull requests by committing with `--sign-off`
+or `-s`, e.g.
 
-Only pull requests from committers that can be verified as having signed the OCA can be accepted.
+```text
+git commit --signoff
+```
 
-### [Pull request process](http://github.com/oracle/wls-deploy/blob/master/CONTRIBUTING.md)
+Only pull requests from committers that can be verified as having signed the OCA
+can be accepted.
 
+## Pull request process
+
+1. Ensure there is an issue created to track and discuss the fix or enhancement
+   you intend to submit.
 1. Fork this repository
-2. Create a branch in your fork to implement the changes. We recommend using the issue number as part of your branch name, e.g. 1234-fixes
-3. Ensure that any documentation is updated with the changes that are required by your fix.
-4. Ensure that any samples are updated if the base image has been changed.
-5. Squash your branch changes into a single commit.
-6. Submit the pull request. Do not leave the pull request blank. Explain exactly what your changes are meant to do and provide simple steps on how to validate your changes. Ensure that you reference the issue you created as well. We will assign someone to review the pull request before it is merged.
+1. Create a branch in your fork to implement the changes. We recommend using
```

Comment:
the numbering messed up here

Reply:
The numbers don't actually matter for a markdown file. You can just use "1" over and over as here.

Reply:
You can validate by looking at the generated file here: https://github.com/oracle/weblogic-deploy-tooling/blob/update-template/CONTRIBUTING.md